src/modules/Dashboard/features/Applet/Schedule/Calendar/EventWrapper/EventWrapper.tsx

```diff
@@ -20,11 +20,43 @@ export const EventWrapper = ({
 
   const tooltipTitle = (
     <>
-      <Box>{`${format(start, DateFormats.Time)} - ${format(end, DateFormats.Time)}`}</Box>
+      <Box>{`${startTime ?? format(start, DateFormats.Time)} - ${
+        endTime ?? format(end, DateFormats.Time)
+      }`}</Box>
       <Box>{title}</Box>
     </>
   );
 
+  const renderEventWrapper = () => {
+    if (timeView && isHiddenInTimeView) {
```

Comment:
I guess it avoid of usage `renderEventWrapper` if you will move early return above into content of the component:
```
 if (timeView && isHiddenInTimeView) {
      return <Box ref={emptyRef} />;
 }
```